docs/topics/opt-in-requirements.md

```diff
@@ -1,15 +1,11 @@
 [//]: # (title: Opt-in requirements)
 
-> The opt-in requirement annotations `@RequiresOptIn` and `@OptIn` are [Experimental](components-stability.md). 
-> They may be dropped or changed at any time. Opt-in is required (see details below).
-> Use them only for evaluation purposes. We appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issues/KT).
->
+> The opt-in requirement annotations `@RequiresOptIn` and `@OptIn` are in [Beta](components-stability.md).
```

Comment:
Why not use our agreed wording for beta warnings (https://docs.google.com/document/d/1zmjZvQHsvkohMZCQTpv64gPDZEpUOQwYqQ-YH601oWc/edit#heading=h.ty9tig7yrvtm)?

Reply:
Thanks, I saved the "See the details..." sentence to clarify the situation for users that can get confused about "Opt-in to opt-in annotations". Addressing [the ticket](https://youtrack.jetbrains.com/issue/KT-49238).